conf/broker.conf

```diff
@@ -1941,10 +1941,17 @@ managedLedgerMaxUnackedRangesToPersistInZooKeeper=-1
 # persist, it will help to reduce the duplicates caused by the ack state that can not be fully persistent.
 dispatcherPauseOnAckStatePersistentEnabled=false
 
-# If enabled, the maximum "acknowledgment holes" will not be limited and "acknowledgment holes" are stored in
-# multiple entries.
+# If enabled, the maximum "acknowledgment holes" (as defined by managedLedgerMaxUnackedRangesToPersist)
+# can be stored in multiple entries, allowing the higher limits.
 persistentUnackedRangesWithMultipleEntriesEnabled=false
 
+# If persistentUnackedRangesWithMultipleEntriesEnabled, this sets maximum entry size for storage in bytes.
+#persistentUnackedRangesMaxEntrySize=1048576
+
+# Set the compression type to use for cursor info.
+# Possible options are NONE, LZ4, ZLIB, ZSTD, SNAPPY
+#cursorInfoCompressionType=NONE
```

Comment:
It's better to remove the comment from the line. The reason for this is that when it's without the comment, it won't require `PULSAR_PREFIX_` with the k8s config hack that Pulsar users.

Please also add the config changes to `standalone.conf`